Live configurator POC (#2353)
mui/mui-toolpad: docs/data/toolpad/live-config/index.md

> # Live Configurators

Reviewer: Sentence case

```suggestion
# Live configurators
```

> <p class="description">Update your datagrid configuration straight in your application</p>

Reviewer: ```suggestion
<p class="description">Update your data grid configuration straight in your application.</p>
```

> ## Live configurators

Reviewer: Looks redundant.

```suggestion
```

The live configurators concept enhances the MUI X components workflow. It renders a playground for your data components right inside your application. Any change you make will immediately be persisted in your project. Toolpad produces performant and production ready code as if you write it by hand.

- **Live editable components in your application**

  Import our generated components and edit them in our browser devtool. Get immediate feedback and see your changes automatically saved in your project folder.

- **Eject any time you want**

  There is no lock-in, you can eject at any time you like. Toolpad will produce readable code for your component. Use it as your starting point for more complex use-cases.

- **Generates production ready code**

  Don't compromise on performance, our generators produce the same optimized code that you would have hand-written. If the result satisfies your use-case

## Demo

Click the edit button to open the live configurator. Hit save when you're done and see the page update.

{{"demo": "BasicExampleGrid.js"}}

> ## installation

Reviewer: ```suggestion
## Installation
```
